manager/integration/tests/backupstore.py

```diff
@@ -170,6 +170,48 @@ def nfs_get_backup_blocks_dir():
     raise NotImplementedError
 
 
+def backupstore_create_file(client, core_api, prefix, file_name, data={}):
```

Comment:
The input args look a little bit weird at first glance. Can we use `(client, core_api, file_path, data={})` instead?
Same below.

Reply:
fixed